src/stylesheets/layouts/_header.scss

```diff
@@ -0,0 +1,136 @@
+@use "../abstracts/" as *;
+
+.header-container {
+    width: 100%;
+    background-color: #d9d9e5;
+
+    .header-top {
+        width: 100%;
+        height: pxToRem(95);
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding: 0 pxToRem(120);
+
+        // logo
+        &__logo {
+            .header-top__logo-item {
+                display: flex;
+                align-items: center;
+                column-gap: 14px;
+
+                .logo {
+                    width: pxToRem(44);
+                    height: pxToRem(44);
+                    border-radius: 50%;
+                    background-color: $color-main;
+                }
+
+                span {
+                    color: $color-main;
+                    font-weight: 600;
+                }
+            }
+        }
+
+        // nav header top
+        &__nav-container {
+            ul {
+                display: flex;
+                column-gap: pxToRem(40);
+            }
+        }
+
+        &__button {
+            span {
+                padding-right: 20px;
+                font-size: 1rem;
+            }
+        }
+    }
+
+    .header-title {
+        width: 100%;
+        height: 758px;
+        // set bg header
+        background-image: linear-gradient(0, white, rgba(204, 204, 216, 0.868)), url(./../assets/img/backgroud.png);
```

Comment:
bg should be placed in the public file

Reply:
> bg should be placed in the public file

Sorry I don't really understand this, can you explain more clearly?